feature-swap-impl/src/main/java/io/novafoundation/nova/feature_swap_impl/presentation/main/SwapSettingsPayload.kt

```diff
@@ -0,0 +1,9 @@
+package io.novafoundation.nova.feature_swap_impl.presentation.main
+
+import android.os.Parcelable
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.ChainAssetId
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.ChainId
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+class SwapSettingsPayload(val chainId: ChainId, val assetId: ChainAssetId) : Parcelable
```

Comment:
`class SwapSettingsPayload(val assetPayload: AssetPayload) : Parcelable`

Reply:
In this case we should move AssetPayload to another module.
I guess we can put it in common module

Reply:
It's a bit complicated to do since AssetPayload is in feature_asset but we cant depend on it

Reply:
We can move it to more general module